core/lib.rs

```diff
@@ -66,23 +69,23 @@ pub use crate::modules::ModuleType;
 pub use crate::modules::NoopModuleLoader;
 pub use crate::normalize_path::normalize_path;
 pub use crate::ops::serialize_op_result;
+pub use crate::ops::to_op_result;
 pub use crate::ops::Op;
 pub use crate::ops::OpAsyncFuture;
 pub use crate::ops::OpCall;
+pub use crate::ops::OpError;
 pub use crate::ops::OpFn;
 pub use crate::ops::OpId;
-pub use crate::ops::OpPayload;
 pub use crate::ops::OpResult;
 pub use crate::ops::OpState;
-pub use crate::ops::OpTable;
 pub use crate::ops::PromiseId;
 pub use crate::ops_builtin::op_close;
 pub use crate::ops_builtin::op_print;
 pub use crate::ops_builtin::op_resources;
-pub use crate::ops_json::op_async;
-pub use crate::ops_json::op_sync;
-pub use crate::ops_json::void_op_async;
-pub use crate::ops_json::void_op_sync;
+pub use crate::ops_builtin::void_op_async;
+pub use crate::ops_builtin::void_op_sync;
+pub use crate::ops_metrics::OpMetrics;
+pub use crate::ops_metrics::OpsTracker;
```

Comment:
Why are these two public?

Reply:
OpMetrics is no longer public. OpsTracker is used by the macro generated code to call `op_state.tracker.track_sync()`